packages/account/src/Components/trading-assessment/trading-assessment-form.tsx

```diff
@@ -17,16 +29,20 @@ const TradingAssessmentForm = ({
     should_move_to_next,
     setSubSectionIndex,
     is_independent_section,
-}) => {
+}: TTradingAssessmentForm) => {
     const [is_section_filled, setIsSectionFilled] = React.useState(false);
     const [should_inform_user, shouldInformUser] = React.useState(false);
+    const [form_data, setFormData] = React.useState<TFormData>({ ...form_value });
```

Comment:
do we need to specify `TFormData` explicitly? it will be inferred from `form_value`, right?
you may also ignore this.